tasks-by-users/AbbasShaikh0304/01-Project/Style.css

```diff
@@ -0,0 +1,9 @@
+.Heading {
+    color: orangered;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    background-image: none;
+}
+.Genre {
+    display: block;
+}
```

Comment:
Where is this css used in the code? I dont see any Heading or Genre classes added in html files.